package.json

```diff
@@ -4,11 +4,12 @@
   "scripts": {
     "prebuild": "rimraf packages/core/dist/ packages/preact/dist",
     "build": "pnpm build:core && pnpm build:preact && pnpm build:react-runtime && pnpm build:react && pnpm build:react-transform",
-    "build:core": "microbundle --raw --cwd packages/core && pnpm postbuild:core",
-    "build:preact": "microbundle --raw --cwd packages/preact && pnpm postbuild:preact",
-    "build:react": "microbundle --raw --cwd packages/react && pnpm postbuild:react",
-    "build:react-runtime": "microbundle --raw --cwd packages/react/runtime && pnpm postbuild:react-runtime",
-    "build:react-transform": "microbundle --no-compress --raw --cwd packages/react-transform",
+    "_build": "microbundle --raw --globals @preact/signals-core=preactSignalsCore",
```

Comment:
Not sure if this is any better tbh; either way I find it to be an unreadable wall of text. At least it reduces the duplication of the `--raw` and `--globals` flags? Happy to revert.